src/components/VideoPlayerPreview/index.tsx

```diff
@@ -12,30 +13,21 @@ import useWindowDimensions from '@hooks/useWindowDimensions';
 import CONST from '@src/CONST';
 import VideoPlayerThumbnail from './VideoPlayerThumbnail';
 
-const propTypes = {
-    videoUrl: PropTypes.string.isRequired,
-
-    videoDimensions: PropTypes.shape({
-        width: PropTypes.number.isRequired,
-        height: PropTypes.number.isRequired,
-    }),
-
-    videoDuration: PropTypes.number,
-
-    thumbnailUrl: PropTypes.string,
-
-    fileName: PropTypes.string.isRequired,
-
-    onShowModalPress: PropTypes.func.isRequired,
+type VideoDimensions = {
+    width: number;
+    height: number;
 };
 
-const defaultProps = {
-    videoDimensions: CONST.VIDEO_PLAYER.DEFAULT_VIDEO_DIMENSIONS,
-    thumbnailUrl: undefined,
-    videoDuration: 0,
+type VideoPlayerPreviewProps = {
+    videoUrl: string;
+    videoDimensions: VideoDimensions;
```

Comment:
```suggestion
    videoDimensions?: VideoDimensions;
```

According to propTypes

Reply:
I was following [Expensify PropTypes Conversion Table](https://github.com/Expensify/App/blob/main/contributingGuides/PROPTYPES_CONVERSION_TABLE.md#dont-rely-on-isrequired)

> **Don't Rely on isRequired**
Regardless of isRequired is present or not on props in PropTypes, read through the component implementation to check if props without isRequired can actually be optional. The use of isRequired is not consistent in the current codebase. Just because isRequired is not present, it does not necessarily mean that the prop is optional.